winui/AutoComplete/Highlighting.md

```diff
@@ -0,0 +1,187 @@
+It highlights the matches that starts with the typed characters in suggestion list.
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfAutoComplete x:Name="autoComplete"                            
+                        DisplayMemberPath="Name"                             
+                        TextHighlightMode="Matched"
+                        TextSearchMode="StartsWith"                           
+                        HighlightedTextForeground="Red"                               
+                        ItemsSource="{Binding Countries}">
+</editors:SfAutoComplete>
+{% endhighlight %}
+{% highlight C# %}
+
+autoComplete.DisplayMemberPath = "Name";
+autoComplete.TextHighlightMode = AutoCompleteTextHighlightMode.Matched;
+autoComplete.TextSearchMode = AutoCompleteTextSearchMode.StartsWith;
+autoComplete.HighlightedTextForeground = Color.Red;
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Searching with one character
+
+![WinUI AutoComplete text highlighting based on begining text one character](Highlighting_images/winui-autocomplete-textsearchmode-startswith-onecharacter.png)
+
+### Searching with more than one character
+
+![WinUI AutoComplete text highlighting based on begining text more than one character](Highlighting_images/winui-autocomplete-textsearchmode-startswith-morethanonecharacter.png)
+
+
+
+
+### **Highlight all occurance of search text**
+It highlights all the matches that contains the typed characters in suggestion list. 
+
+
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfAutoComplete     
+                x:Name="autoComplete"                      
+                DisplayMemberPath="Name”                              
+                TextHighlightMode="Matched"
+                TextSearchMode="Contains"                           
+                HighlightedTextForeground="Red"                               
+                ItemsSource="{Binding Wonders}">
+</editors:SfAutoComplete>
+
+{% endhighlight %}
+{% highlight C# %}
+
+        autoComplete.DisplayMemberPath = "Name";
+        autoComplete.TextHighlightMode = AutoCompleteTextHighlightMode.Matched;
+        autoComplete.TextSearchMode = AutoCompleteTextSearchMode.Contains;
+        autoComplete.HighlightedTextForeground = Color.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Searching with one character
+![WinUI AutoComplete text highlighting based on contains one character](Highlighting_images/winui-autocomplete-textsearchmode-contains-one.png)
+
+### Searching with more than one character
+
+![WinUI AutoComplete text highlighting based on contains more than one character](Highlighting_images/winui-autocomplete-textsearchmode-contains-morethanone.png)
+
+
+## **Highlighting search text mode** 
+ The SfAutoComplete provides the following modes.
+* **None** - This mode doesnot highlight any text.
+* **Matched** - This mode highlights the text that matches the user input.
+* **Unmatched** - This mode highlights the text that doesnot matches the user input.
```

Comment:
* **Unmatched** - This mode highlights the text that does not match the user input.